Added mobile number validation

VibhorCodecianGupta opened the pull request:
fixes #376

championswimmer commented on src/routers/pages/users/index.js:
```diff
@@ -62,13 +63,23 @@ router.post('/:id/edit',
                 lastname: req.body.lastname,
                 gender:req.body.gender,
                 email: req.body.email,
-                mobile_number: req.body.mobile_number,
```
rather send an error back to user that the phone number is not valid instead of creating user with null phone number

championswimmer commented:
This pull request **introduces 1 alert** when merging f9bc93a3d5dbf878c889cc38e3fe102f5af97a5f into 0bd00efb100e401346bd624ebdfe2991d87b6c9b - [view on LGTM.com](https://lgtm.com/projects/g/coding-blocks/oneauth/rev/pr-dd508e26907f647bac88c286e0055e456c5de678)

**new alerts:**

* 1 for Server\-side URL redirect

---

*Comment posted by [LGTM.com](https://lgtm.com)*

VibhorCodecianGupta commented:
@championswimmer done completely. Fixed the error redirect as well, it was redirecting to a 404. In case of an invalid phone number, the user(admin) is redirected to the edit page itself. Also, the Google library parse method returns a humungous object, the phone number has to be extracted. Wasn't there before, added now.

championswimmer commented:
This pull request **introduces 2 alerts** when merging 6e01d847bd8cf3bd227a87a9c1da1d5e2b3f54cc into 0bd00efb100e401346bd624ebdfe2991d87b6c9b - [view on LGTM.com](https://lgtm.com/projects/g/coding-blocks/oneauth/rev/pr-f2f8e440cc7e42e4f70a6983b3bdc7c609c6b2b7)

**new alerts:**

* 2 for Server\-side URL redirect

---

*Comment posted by [LGTM.com](https://lgtm.com)*

championswimmer commented on src/routers/pages/users/index.js:
```diff
@@ -70,7 +70,8 @@ router.post('/:id/edit',
       } else {
-          mobile_number = null
+          req.flash('error', 'Please enter a valid number!')
+          res.redirect('../' + req.params.id);
```
`return res.send()` so that code execution doesn't proceed

VibhorCodecianGupta replied:
Done @championswimmer

championswimmer commented:
This pull request **introduces 2 alerts** when merging 74932ac0b9488835cdf4313d0366505758286968 into 0bd00efb100e401346bd624ebdfe2991d87b6c9b - [view on LGTM.com](https://lgtm.com/projects/g/coding-blocks/oneauth/rev/pr-444404dfedcca4a46abcbfcad1ca91cd53dae263)

**new alerts:**

* 2 for Server\-side URL redirect

---

*Comment posted by [LGTM.com](https://lgtm.com)*
